shanoir-ng-datasets/src/main/java/org/shanoir/ng/bids/service/BIDSServiceImpl.java

```diff
@@ -201,8 +219,10 @@ public void deleteBidsFolder(Long studyId, String studyName) {
 	 * @throws IOException
 	 */
 	@Override
-	public File exportAsBids(final Long studyId, final String studyName) throws IOException {
+	public File exportAsBids(final Long studyId, String studyName) throws IOException {
 		// Get folder
+		studyName = studyName.replaceAll(" ", "");
```

Comment:
I think you can do that in one line using regex :
`studyName = studyName.replaceAll("[ _] ", ""); `